src/app/modules/group/components/group-log-view/group-log-view.component.html

```diff
@@ -35,15 +35,21 @@
           pTemplate="body"
```

Comment:
> The breadcrumb are shown as "via ItemA1/ItemA2" with appropriate ellipses when the titles are too long.

"Via [breadcrumb]" -> the "via" is missing

Reply:
Fixed